css/templates/travel/elements/_border.css

```diff
@@ -0,0 +1,8 @@
+/*
+ * border
+ *
+ */
+
+ .travel-border-gray {
```

Comment:
dont create files for one off classes. also fix the name not a good name

Reply:
if you want to change the border color for this template there is already a variable to override